certif/tests/integration/components/enrolled-candidates_test.js

```diff
@@ -191,6 +192,68 @@ module('Integration | Component | enrolled-candidates', function (hooks) {
       .hasClass(DELETE_BUTTON_SELECTOR);
   });
 
+  module('when feature toggle FT_CERTIFICATION_BILLING is enabled and certification center is not SCO', function () {
+    test('it displays candidate billing information', async function (assert) {
+      // given
+      this.set('shouldDisplayPaymentOptions', true);
+      const candidate = _buildCertificationCandidate({
+        billingMode: 'Prepayée',
+        prepaymentCode: 'CODE01',
+      });
+
+      const certificationCandidate = store.createRecord('certification-candidate', candidate);
+
+      const certificationCandidates = [certificationCandidate];
+
+      this.set('certificationCandidates', certificationCandidates);
+
+      // when
+      await render(hbs`
+          <EnrolledCandidates
+            @sessionId="1"
+            @certificationCandidates={{certificationCandidates}}
+            @shouldDisplayPaymentOptions={{shouldDisplayPaymentOptions}}
+            >
+          </EnrolledCandidates>
+        `);
+
+      // then
+      assert.contains('Tarification part Pix');
+      assert.dom(`[data-test-id=${PAYMENT_OPTION_SELECTOR}${candidate.id}]`).hasText('Prepayée CODE01');
```

Comment:
Question: 
il y a t'il une alternative à data-test-id?

Reply:
Compliqué car ici le `data-test-id` permet de vérifier que le texte apparaît dans la bonne colonne et la bonne ligne.

Avec testing-library on pourrait faire :

```suggestion
      assert.dom(screen.getByText('Prepayée CODE01')).exists();
```

Mais dans ce cas on vérifie juste que le texte apparaît et pas l'endroit où il est affiché.